concepts/numbers/introduction.md

```diff
@@ -1,11 +1,5 @@
 # Introduction
 
-TODO: the content below is copied from the exercise introduction and probably needs rewriting to a proper concept introduction
```

Comment:
This TODO is not really addressed. There are still mentions of the exercise in the section below, but this material is supposed to be exercise agnostic.

Reply:
I opened a separate issue for this #1562 - I agree with you, but I think this change is out of the scope of this PR as it requires thinking about `about.md` too.